### PR TITLE
Btc004

### PR DESCRIPTION
**Pull request by Ozleblebici**
Ticket: www.atlasianJira.com/BTC004
My ticket is ready for review. 
There is no change to the existing framework, just the basic 2 new tests uploaded. 

Thank you,
Talha
Here is successful screenshot for test:
image

### CODE REVIEW

File: src/test/java/BTC004Tests/Test2/Test2.java [hunk 1]
```diff
@@ -0,0 +1,15 @@
+package BTC004Tests.Test2;
+
+public class Test2 {
+
+    public static void main(String[] args) {
+        // code 1
+        // verify (exp , act);
+        // verify (exp , act);
```
Comment (Kkmyrat) [inline src/test/java/BTC004Tests/Test2/Test2.java]: this is an extra duplicated verify method, pls delete it

File: src/test/java/BTC004Tests/Test1/Test1.java [hunk 2]
```diff
@@ -0,0 +1,9 @@
+package BTC004Tests.Test1;
+
+public class Test1 {
+
+    // code1
+    // code2
```
Comment (Kkmyrat) [inline src/test/java/BTC004Tests/Test1/Test1.java:6]: is this code necessary for this test automation? I guess need to run code3

Comment (Ozleblebici) [reply src/test/java/BTC004Tests/Test1/Test1.java:6]: If u look at the ticket scenarios details u will see that need to run "code2"

File: src/test/java/BTC004Tests/Test2/Test2.java (hunk 1 above)
Comment (Ozleblebici) [reply src/test/java/BTC004Tests/Test2/Test2.java]: Updated in [193d561](https://github.com/euroTech-sdet/Batch4_GitPractice/pull/1/commits/193d5611c02a19f29fa297f558ae122ceab4885c) commit

**Review by Kkmyrat — APPROVED** [review]
All looks ok for me,
Thank you Talha

File: src/test/java/BTC004Tests/Test1/Test1.java (hunk 2 above)
Comment (halilarslan1999) [reply src/test/java/BTC004Tests/Test1/Test1.java:6]: I agree wıth you @Ozleblebici

**Review by halilarslan1999 — APPROVED** [review]
All looks for for me as well

**Pull request merged**